tests/pyccel/project_rel_imports/runtest.py

```diff
@@ -1,5 +1,7 @@
 # pylint: disable=missing-function-docstring, missing-module-docstring
 from project.folder2.mod3 import one_hundred_plus_sum_to_n_squared
+from project.folder1.mod1 import add2
 
 if __name__ == '__main__':
     print(one_hundred_plus_sum_to_n_squared(4))
+    print(add2(1.0, 4.0))
```

Comment:
Issue #2430 says that "The call to `insert_function` leads to the implementations being overwritten leaving only the final implementation." In fact here we are using the second and last implementation of `add2`, which works on float numbers. So I expect this unit test to work even before this PR...

Reply:
The first implementation is used in `project/folder1/mod2.py` I added this to make sure that both implementations were being tested rather than to trigger the test